packages/unity-interface/BrowserInterface.ts

```diff
@@ -686,6 +686,8 @@ export class BrowserInterface {
     deployScene(data)
       .then(() => {
         deploymentResult = { ok: true }
+        if (!data.reloadSingleScene) void invalidateAllScenes()
```

Comment:
Please revert the order of the if and remove the ! for clarity

Reply:
In which cases is `reloadSingleScene=false` needed?

Reply:
I will! 

It is needed when you are publishing inside the editor, if you are inside the editor you don't need to reload a single scene since they will be loaded when you exit, however if you are publishing outside the editor ( from the panel) you will need to reload that scene